Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -89,9 +89,32 @@ @interface AVSampleBufferDisplayLayer (Staging_100128644)
     , m_logIdentifier(player->mediaPlayerLogIdentifier())
     , m_videoLayerManager(makeUnique<VideoLayerManagerObjC>(m_logger, m_logIdentifier))
     , m_listener(WebAVSampleBufferListener::create(*this))
+    , m_seekTimer(*this, &MediaPlayerPrivateWebM::seekInternal)
 {
     ALWAYS_LOG(LOGIDENTIFIER);
     m_parser->setLogger(m_logger, m_logIdentifier);
+
+    // addPeriodicTimeObserverForInterval: throws an exception if you pass a non-numeric CMTime, so just use
+    // an arbitrarily large time value of once an hour:
+    __block WeakPtr weakThis { *this };
+    m_timeJumpedObserver = [m_synchronizer addPeriodicTimeObserverForInterval:PAL::toCMTime(MediaTime::createWithDouble(3600)) queue:dispatch_get_main_queue() usingBlock:^(CMTime time) {
+#if LOG_DISABLED
+        UNUSED_PARAM(time);
+#endif
+        // FIXME: Remove the below once <rdar://problem/15798050> is fixed.
```

Comment:
It seems fixed in 2014, I am unclear what is needed here.

Reply:
This change replicate the well tested MediaPlayerPrivateMediaSourceAVFObjC 

Idea is to replicate the code that is known to work.

I don't take the assumption that what was currently there was correct, because it wasn't.